apps/speed.c

```diff
@@ -897,37 +901,36 @@ static int EVP_Update_loop(void *args)
         EVP_EncryptFinal_ex(ctx, buf, &outl);
     return count;
 }
-/*
- * CCM does not support streaming. For the purpose of performance measurement,
- * each message is encrypted using the same (key,iv)-pair. Do not use this
- * code in your application.
- */
```

Comment:
I removed this warning because it would be much needed in other places also. Having it *only* here kind of seems to imply that all other speed tool code is "best practice", which is not true e.g., just resetting counter w/o chaning key/iv in generig update loop ...